### PR TITLE
added tooltip for thread title

### PR DESCRIPTION
**Pull request by mkbeefcake**
Fixed) Forum: tooltip to show full thread title #3273

### CODE REVIEW

File: packages/ui/src/forum/components/threads/ThreadListItem.tsx [hunk 1]
```diff
@@ -35,10 +36,12 @@ export const ThreadListItem = ({ thread, isArchive }: ThreadListItemProps) => {
     >
       {thread.isSticky && <ThreadPinIcon />}
 
-      <Thread>
-        <TextBig bold>{thread.title}</TextBig>
-        {thread.tags.length > 0 && <ThreadTags tags={thread.tags} />}
-      </Thread>
+      <Tooltip tooltipText={thread.title}>
+        <Thread>
+          <TextBig bold>{thread.title}</TextBig>
```
Comment (traumschule) [inline packages/ui/src/forum/components/threads/ThreadListItem.tsx:41]: > "Radosław" add this line of code to the title component for truncation:
> truncateLines={2}

**Review by traumschule — CHANGES_REQUESTED** [review]
LGTM. From the image this is already truncated. CSS or how does it work?
thread-title

File: packages/ui/src/forum/components/threads/ThreadListItem.tsx (hunk 1 above)
Comment (mkbeefcake) [reply packages/ui/src/forum/components/threads/ThreadListItem.tsx:41]: image
This is the CSS which is for truncated features (I just checked that it already fixed)

Comment (WRadoslaw) [reply packages/ui/src/forum/components/threads/ThreadListItem.tsx:41]: `FullDots` will truncate first line only. `truncateLines` props on all `Text` components allows to pick number of line which will be truncated. `truncateLines={1}` should work as well

**Review by WRadoslaw — APPROVED** [review]

**Pull request merged**